debian/install

```diff
@@ -1,2 +1,3 @@
-debian/homeserver.yaml etc/matrix-synapse
-debian/log.yaml etc/matrix-synapse
+../homeserver.yaml etc/matrix-synapse
```

Comment:
are these not still inside the `debian` directory?

Reply:
so, the old system would copy the entire tree into the source dir, and so it would be. This one only extracts a tarball, so instead of getting it from the 'source' we need to go back down into the original debian directory and fish it out.

Reply:
I think I fixed it!